harvester/core/fixtures/datasets-history.json

```diff
@@ -1,992 +1,871 @@
+      "modified_at": "2020-02-17T12:24:52.119Z",
+      "name": "edusources",
+      "identifier": null,
+      "referee": "external_id",
+      "dataset_version": 1
+    }
+  },
+  {
+    "model": "core.harvestsource",
+    "pk": 1,
+    "fields": {
+      "name": "SURF Sharekit",
+      "repository": "sharekit.SharekitMetadataHarvest",
+      "spec": "edusources",
+      "created_at": "2020-02-06T17:13:15.417Z",
+      "modified_at": "2020-02-06T17:13:15.417Z"
+    }
+  },
+  {
+    "model": "core.harvestsource",
+    "pk": 2,
+    "fields": {
+      "name": "Edurep Delen",
+      "repository": "sharekit.SharekitMetadataHarvest",
+      "spec": "edurep_delen",
+      "created_at": "2019-12-06T10:58:40.757Z",
+      "modified_at": "2019-12-19T15:00:18.560Z"
+    }
+  },
+  {
+    "model": "core.harvest",
+    "pk": 1,
+    "fields": {
+      "source": 1,
+      "dataset": 19,
+      "latest_update_at": "1970-01-01T00:00:00Z",
+      "harvested_at": "2020-02-10T13:08:39.315Z",
+      "purge_after": null,
+      "is_syncing": false,
+      "stage": "Complete"
+    }
+  },
+  {
+    "model": "core.harvest",
+    "pk": 2,
+    "fields": {
+      "source": 2,
+      "dataset": 19,
+      "latest_update_at": "1970-01-01T00:00:00Z",
+      "harvested_at": "2020-02-10T13:08:39.315Z",
+      "purge_after": null,
+      "is_syncing": false,
+      "stage": "Video"
+    }
+  },
+  {
+    "model": "core.extension",
+    "pk": "5af0e26f-c4d2-4ddd-94ab-7dd0bd531751",
+    "fields": {
+      "created_at": "2021-02-17T10:53:24.387Z",
+      "modified_at": "2021-02-17T10:53:24.387Z",
+      "collection": null,
+      "identity": null,
+      "reference": "5af0e26f-c4d2-4ddd-94ab-7dd0bd531751",
+      "is_addition": false,
+      "properties": {
+        "external_id": "5af0e26f-c4d2-4ddd-94ab-7dd0bd531751",
+        "authors": [{ "name": "The Extension Man" }],
+        "keywords": [{ "label": "exercise" }, { "label": "extended" }],
+        "themes": [{ "label": "theme" }, { "label": "extended" }],
+        "children": ["part", "child"],
+        "parents": ["parent"],
+        "parties": [{ "name": "The Extension Party" }]
+      }
+    }
+  },
+  {
+    "model": "core.extension",
+    "pk": "custom-extension",
+    "fields": {
+      "created_at": "2021-02-17T10:53:24.387Z",
+      "modified_at": "2021-02-17T10:53:24.387Z",
+      "collection": null,
+      "identity": null,
+      "reference": "custom-extension",
+      "is_addition": true,
+      "properties": {
+        "state": "active",
+        "external_id": "custom-extension",
+        "title": "New! New! New! Extended titles!",
+        "authors": [{ "name": "The Extension Man" }],
+        "keywords": [{ "label": "exercise" }, { "label": "extended" }],
+        "themes": [{ "label": "theme" }, { "label": "extended" }],
+        "children": ["part", "child"],
+        "parents": ["parent"],
+        "parties": [{ "name": "The Extension Party" }],
+        "consortium": null
```

Comment:
Gevonden, deze moet er niet in zitten. Dan falen de tests. Door consortium toe te voegen aan to_search van Extension model fix je het op de correcte manier @naftalivanderloon